lotus/sem_ops/sem_join.py

```diff
@@ -182,6 +198,40 @@ def sem_join_cascade(
     num_helper = len(helper_high_conf)
     num_large = len(helper_low_conf)
 
+    if safe_mode:
+        sample_docs1 = task_instructions.df2multimodal_info(l1.to_frame(col1_label), [col1_label])
+        sample_docs2 = task_instructions.df2multimodal_info(l2.to_frame(col2_label), [col2_label])
+        sample_helper_doc = task_instructions.merge_multimodal_info([sample_docs1[0]], sample_docs2)
+
+        estimated_tokens_helper = lotus.settings.helper_lm.count_tokens(
```

Comment:
This implementation is not correct since `helper_lm` isn't used for `join_cascade`. We could just leave the safe_mode as a TODO here for now and add it in another PR